certification_organism/data/knowledge_article.xml

```diff
@@ -1,15 +1,15 @@
 <?xml version='1.0' encoding='UTF-8'?>
 <odoo>
```

Comment:
@pimodoo Shouldn't this be no-update, to avoid breaking user's article if they change it themselves?
```suggestion
<odoo noupdate="1">
```

Reply:
Apparently, no.